### PR TITLE
update to 2.1.1

### PR DESCRIPTION
**Pull request by cbouss**
Changes:
- update to 2.1.1 (new feedstock)

https://github.com/oracle/python-oracledb/tree/v2.1.1

### CODE REVIEW

**Review by JeanChristopheMorinPerso — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -38,8 +38,11 @@ test:
 about:
   home: https://oracle.github.io/python-oracledb
   summary: Python interface to Oracle Database
+  description: |
+    python-oracledb is a Python programming language extension module allowing Python programs to connect to Oracle Database. 
+    It is the renamed, new major release of the popular cx_Oracle driver.
   dev_url: https://github.com/oracle/python-oracledb
-  doc_url: https://python-oracledb.readthedocs.io/en/latest/
+  doc_url: https://python-oracledb.readthedocs.io
   license: Apache-2.0
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:46]:
```suggestion
  license: Apache-2.0 OR UPL-1.0
```
See https://github.com/oracle/python-oracledb/blob/v2.1.1/LICENSE.txt#L3-L6

Comment (cbouss) [reply recipe/meta.yaml]: thanks

File: recipe/meta.yaml [hunk 2]
```diff
@@ -38,9 +38,12 @@ test:
...
+    python-oracledb is a Python programming language extension module allowing Python programs to connect to Oracle Database. 
+    It is the renamed, new major release of the popular cx_Oracle driver.
   dev_url: https://github.com/oracle/python-oracledb
-  doc_url: https://python-oracledb.readthedocs.io/en/latest/
-  license: Apache-2.0
+  doc_url: https://python-oracledb.readthedocs.io
+  license: Apache-2.0 OR UPL-1.0
   license_family: Apache
   license_file:
     - LICENSE.txt
```
Comment (Jrice1317) [inline recipe/meta.yaml:49]:
```suggestion
    - LICENSE.txt
    - THIRD_PARTY_LICENSES.txt
```

**Review by Jrice1317 — APPROVED** [review]
Not sure if it's a blocker or not, but I've added the third license as a comment JIC.

File: recipe/meta.yaml (hunk 2 above)
Comment (cbouss) [reply recipe/meta.yaml:49]: The content of the third party license file lists license of dependencies (cython, cryptography). It does seem to me that this applies to code provided with oracledb. And it doesn't seem to be up-to-date. I'll leave it out.

**Pull request merged**